core/controllers/pages_test.py

```diff
@@ -47,3 +49,19 @@ def test_maintenance_page(self):
             self.assertIn(
                 'Oppia is currently being upgraded, and the site should be up',
                 response.body)
+
+    def test_maintenance_mode_url(self):
+        import main
```

Comment:
Why do you need to reimport main here?

Reply:
We need to reimport main here else we will get an error:
```python
UnboundLocalError: local variable 'main' referenced before assignment
```

This is because we are reimporting main again below

Reply:
Please add a comment to explain.

Reply:
Done